stellarphot/settings/settings_files.py

```diff
@@ -352,8 +352,32 @@ def save(self, settings):
         ----------
         settings : PhotometrySettings
             The settings to save.
+
+        update : bool, optional
+            If True, the settings are updated -- see Note below for more description.
+            If False, the settings are overwritten.
+
+        Notes
+        -----
+
+        If ``update`` is True, then the settings are updated. This means that if the
```

Comment:
So should it be an error to have update `True` when the settings are full settings?  Is there anything actually wrong with updating full settings?  Because I understand the logic below, full settings will not get updated.

Reply:
Correct, full settings would override anything saved on disk.